src/main/java/book/store/model/Book.java

```diff
@@ -0,0 +1,34 @@
+package book.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities

```suggestion
@Table(name...)
@Getter
@Setter
```

Reply:
Please tell me. Why not use it. Is that what Bogdan was doing in the video?